src/runtime_src/core/tools/xbmgmt2/SubCmdExamine.cpp

```diff
@@ -121,6 +121,21 @@ SubCmdExamine::execute(const SubCmdOptions& _options) const
     return;
   }
 
+  // Determine report level
+  if (devices.size() == 0 && reportNames.size() == 0)
+    reportNames.push_back("host");
+
+  if (devices.size() != 0 && reportNames.size() == 0) {
+    reportNames.push_back("platform");
+  }
+
+  // Determine default values
+  if (devices.size() == 0)
+    devices.push_back("all");
+
+  if (reportNames.size() == 0)
+    reportNames.push_back("host");
```

Comment:
warning: use emplace_back instead of push_back [hicpp-use-emplace]

```cpp
reportNames.push_back("host");
                ^
note: this fix will not be applied because it overlaps with another fix
```